node/impl/full/gas.go

```diff
@@ -315,6 +315,8 @@ func gasEstimateGasLimit(
 	msg.GasFeeCap = big.Zero()
 	msg.GasPremium = big.Zero()
 
+	log.Infof("gasEstimateGasLimit: msg.GasLimit set to %d, buildconstants.BlockGasLimit: %d", msg.GasLimit, buildconstants.BlockGasLimit)
+
```

Comment:
```suggestion
```

I think this is going to be far too noisy running an operational node

Reply:
@rvagg Is there a way to only run this on CI ?

Reply:
I guess you could check for the `CI` env var but that seems a little excessive for this path. If you really want it you could make it a Debug, but itests hard-wire log level unfortunately so you wouldn't even see it then.
Does this add much value now that you've addressed the problem? Or do you have enough uncertainty about the fix that you want to keep seeing this?